### ISSUE TITLE
[FEATURE] - Option to amend a flight whether its Vatsim or Offline etc during the flight

### ISSUE THREAD

**Issue by Haskins91** [invernyx/smartcars-3-bugs#325]
### Is your feature request related to a problem? Please describe.

At present when choosing a flight and starting, the option for the Vatsim/Offline status it is easy to miss as its hidden away in a drop down menu. 

### Describe the solution you'd like

Can we have an option to amend a flight whether its Vatsim or Offline etc during the flight.

Also have the option for Vatsim/Offline more visible on the despatch page before starting a flight.

Smartcars 2 had the option on the main dispatch page.

### Describe alternatives you've considered

N/A

### Additional context

N/A

**Comment by Thien42** [comment]
Duplicate of https://github.com/invernyx/smartcars-3-bugs/issues/201
